configs/CSC-LUMI-HPECray-zen3-MI250X-Slingshot/variables.yaml

```diff
@@ -6,12 +6,13 @@
 variables:
   gtl_flag: ''  # to be overwritten by tests that need GTL
   rocm_arch: 'gfx90a'
-  batch_time: '02:00'
-  mpi_command: 'srun -N {n_nodes} -n {n_ranks}'
-  batch_submit: 'sbatch {execute_experiment}'
-  batch_nodes: '#SBATCH -N {n_nodes}'
-  batch_ranks: '#SBATCH -n {n_ranks}'
-  batch_timeout: '#SBATCH -t {batch_time}:00'
-  cpu_partition: '#SBATCH -p small'
-  gpu_partition: '#SBATCH -p small-g' 
+  timeout: '120'
+  scheduler: "slurm"
```

Comment:
I think we should preface scheduler-related names with "batch_", so "timeout" would become "batch_timeout".

We need at least 2 more scheduler variables:
- batch_partition_name   (on LC, this would be 'pbatch')
- batch_bank (on LC, lets set this to 'guest')

Ideally, the user should be able to overwrite the latter two from their experiments script - is that possible?

Reply:
> We need at least 2 more scheduler variables:
> batch_partition_name (on LC, this would be 'pbatch')

I used the term "queue" for this (i.e. I translate the `queue` variable to `-p` for `srun`, if `queue` was set), do you want it to in fact be `batch_partition_name`? Or would `batch_queue` work? (any are fine with me, I'm just curious which is the more general term)

> batch_bank (on LC, lets set this to 'guest')

Does this translate to `-A` for `srun`? I don't need to know all the translations, but I'm not clear where to figure out what they are (and I assume we'll want to be able to do this for all the schedulers)

Reply:
> Ideally, the user should be able to overwrite the latter two from their experiments script - is that possible?

You mean a customized `execute_experiment.tpl`? 

* FWIW, once I fold in the logic to combine multiple `variables.yaml` files, the appropriate `*_instructions` function can turn `batch_partition/bank` into arguments for the system, and there could be a single file associated with the user that sets these (and benchpark could combine it with each the system `variables.yaml`)
* But I could also add the option for a customized `execute_experiment.tpl` file (e.g. if there's an option that should be applied for all users of the system)

I think either/both of these would be good follow-up PRs (I want to add the understanding of these variables here, but split up the configuration later).

Reply:
yes, lets go with:
batch queue (-p for srun, pbatch as the default value)
batch bank (-A for srun, guest as the default value on LC)